crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs

```diff
@@ -404,7 +404,7 @@ pub(crate) fn argument_simple_defaults(checker: &mut Checker, args: &Arguments)
 
                         if checker.patch(diagnostic.kind.rule()) {
                             #[allow(deprecated)]
-                            diagnostic.set_fix(Fix::unspecified(Edit::range_replacement(
+                            diagnostic.set_fix(Fix::suggested(Edit::range_replacement(
```

Comment:
Applying the fix suggested by the "argument_simple_defaults" lint rule and using simple default values is generally considered safe and recommended. However, it's important to consider the specific requirements and design of the code before applying this rule. In some cases, using mutable default values may be necessary or desired hence I went with suggested here